Framework/src/TaskRunner.cxx

```diff
@@ -149,7 +149,8 @@ void TaskRunner::init(InitContext& iCtx)
 
   // setup timekeeping
   mDeploymentMode = DefaultsHelpers::deploymentMode();
-  mTimekeeper = TimekeeperFactory::create(mDeploymentMode, mTaskConfig.cycleDurations.back().first * 1000);
```

Comment:
are  you guaranteed that mTaskConfig.cycleDurations is not empty?

Reply:
Thanks, something would have to go very wrong. The configuration goes through `TaskRunnerFactory` which does the input sanitization.